docs/_newsfragments/2167.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Resources without responder methods now emit a ``UserWarning`` for non-suffix responders.
```

Comment:
We could expand the newsfragment a bit, telling a story what the inconsistency was, and how it was addressed.